### PR TITLE
Create Teams page

### PR DESCRIPTION
**Pull request by JjuliaP**
Create Team List page
Create Team Item page

## JIRA

* [Main JIRA ticket](http://ssu-jira.softserveinc.com/browse/LVPYTP-97)

## Code reviewers

- [ ] @dmytrorevak
- [ ] @IvanAndrukh
- [ ] @JjuliaP
- [ ] @makarman
- [ ] @noobhunters
- [ ] @Silent777
- [ ] @Staryj812

### Second Level Review

- [ ] @ssOleg
- [ ] @lhalam

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: eventually/static/src/containers/teamList/TeamItem.js [hunk 1]
```diff
@@ -0,0 +1,114 @@
+import React from 'react';
+import RaisedButton from 'material-ui/RaisedButton';
+import Subheader from 'material-ui/Subheader';
+import EditTeamDialog from './EditTeamDialog';
+import {getImageUrl} from '../../helper';
+import {teamServiceGet} from './teamService';
+const styles = {
```
Comment (redmyt) [inline eventually/static/src/containers/teamList/TeamItem.js]: add new line

File: eventually/static/src/containers/teamList/EditTeamDialog.js [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+                        defaultValue={this.state.description}
+                        fullWidth={true}
+                        onChange={this.handleDescription}
+                    /><br />
+                    <FileUpload updateImageNameInDb={this.uploadImage}/>
+                </div>
+            </Dialog>
+        )
+    }
+}
```
Comment (Silent777) [inline eventually/static/src/containers/teamList/EditTeamDialog.js]: add endline

File: eventually/static/src/containers/teamList/TeamItem.js [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+                    name = {this.props.name}
+                    description = {this.props.description}
+                    image = {this.props.image}
+                    updateItem = {this.props.updateItem}
+                    id = {this.props.id}
+                />
+            </div>
+        );
+    }
+}
```
Comment (Silent777) [inline eventually/static/src/containers/teamList/TeamItem.js]: add endline

File: eventually/static/src/containers/teamList/TeamList.js [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+                        image={team.image}
+                        members={team.members_id.length}
+                        updateItem={this.updateTeamItem}
+                        listOfMembers={team.members_id}
+                    />
+                ))}
+            </div>
+        );
+    }
+}
```
Comment (Silent777) [inline eventually/static/src/containers/teamList/TeamList.js]: add endline

File: eventually/static/src/helper/helper.js [hunk 5]
```diff
@@ -0,0 +1,4 @@
+export const s3Root='https://s3.eu-west-2.amazonaws.com/eventually-photos/';
+export function getImageUrl(imageName){
+    return s3Root + imageName;
+}
```
Comment (Silent777) [inline eventually/static/src/helper/helper.js]: add endline

File: eventually/static/src/helper/index.js [hunk 6]
```diff
@@ -0,0 +1 @@
+export {getImageUrl} from './helper';
```
Comment (Silent777) [inline eventually/static/src/helper/index.js]: add endline

**Review by Silent777 — APPROVED** [review]

**Pull request merged**